src/chart/AccessibilityManager.tsx

```diff
@@ -0,0 +1,98 @@
+import { LayoutType } from '../util/types';
+
+interface ContainerOffset {
+  top: number;
+  left: number;
+}
+
+/* eslint-disable no-underscore-dangle */
+export class AccessibilityManager {
+  private _coordinateList: any[] = [];
+
+  private _activeIndex = 0;
```

Comment:
The activeIndex having a default of 0 is certainly coming from the categorical chart. Given that it make sense both have the same default, how about making sure that they do, by extracting a named constant and importing in both places? 

```
export const DEFAULT_ACTIVE_INDEX = 0
```